iocage/iocage.8

```diff
@@ -0,0 +1,1646 @@
+.Ar TEXT
+.Op Fl -root-dir | d
+.Ar TEXT
+.Op Fl -server | s
+.Ar TEXT
+.Op Fl -update | U
+.Op Fl -user | u
+.Ar TEXT
+.Op Fl -verify | V
+.Op Cm release Ns = Ns Ar RELEASE <REVIEW>
+.Op Cm ftphost Ns = Ns Ar ftp.hostname.org <REVIEW>
+.Op Cm ftpdir Ns = Ns Ar /dir/ <REVIEW>
+.Op Cm ftpfiles Ns = Ns Ar "base.txz doc.txz lib32.txz src.txz" <REVIEW>
+.\" == FSTAB ==
+.Nm
+.Cm fstab
+.Op Fl -add | a
+.Op Fl -edit | e
+.Op Fl -remove | r
+.Ar JAIL
+.Op Ar FSTAB_STRING
+.\" == GET ==
+.Nm
+.Cm get
+.Op Fl -all | a
+.Op Fl -header | H | h
+.Op Fl -plugin | P
+.Op Fl -pool | p
+.Op Fl -recursive | r
+.Ar PROPERTY
+.Op Ar all | UUID | TAG
+.\" == IMPORT ==
+.Nm
+.Cm import
+.Ar JAIL
+.Op Ar Jail_value
+.\" == LIST ==
+.Nm
+.Cm list
+.Op Fl -base | -release | b | r
+.Op Fl -header | H | h
+.Op Fl -http | h
+.Op Fl -long | l
+.Op Fl -plugins | P
+.Op Fl -remote | R
+.Op Fl -template | t
+.\" == MIGRATE ==
+.Nm
+.Cm migrate
+.Op Fl -delete | d
+.Op Fl -force | f
+.\" == PKG ==
+.Nm
+.Cm pkg
+.Ar JAIL
+.Ar COMMAND
+.\" == RESTART ==
+.Nm
+.Cm restart
+.Op Fl -soft | s
+.Op Ar UUID | TAG
+.\" == ROLLBACK ==
+.Nm
+.Cm rollback
+.Op Fl -force | f
+.Op Fl -name | n
+.Ar SNAPSHOT
+.Op Ar UUID | TAG@snapshotname
+.\" == SET ==
+.Nm
+.Cm set
+.Op Fl -plugin | P
+.Ar KEY
+.Ar property_value
+.Op Ar UUID | TAG
+.\" == SNAPLIST ==
+.Nm
+.Cm snaplist
+.Op Fl -header | H | h
+.Op Ar UUID | TAG
+.\" == SNAPREMOVE ==
+.Nm
+.Cm snapremove
+.Op Fl -name | n
+.Op Ar UUID | TAG@snapshotname | ALL
+.\" == SNAPSHOT ==
+.Nm
+.Cm snapshot
+.Op Fl -name | n
+.Op Ar UUID | TAG | TAG@snapshotname
+.\" == START ==
+.Nm
+.Cm start
+.Op Fl -rc
+.Op Ar UUID | TAG
+.\" == STOP ==
+.Nm
+.Cm stop
+.Op Fl -rc
+.Op Ar UUID | TAG
+.\" == UPDATE ==
+.Nm
+.Cm update
+.Op Ar UUID | TAG
+.\" == UPGRADE ==
+.Nm
+.Cm upgrade
+.Op Fl -release | r
+.Ar RELEASE
+.Op Ar UUID | TAG
+.Sh DESCRIPTION
+.Nm
+is a system administration tool designed to simplify jail management
+tasks.
+It abstracts out the management of ZFS-backed jails running VNET or
+shared IP networking.
+.Pp
+Both shared IP jails and VNET jails are supported.
+.Pp
+Each jail has a unique ID (UUID) which is automatically generated at
+creation time.
+Using the UUID as a jail identifier makes it possible to replicate a
+jail in a distributed environment with greater flexibility.
+This also eliminates potential naming clashes on large scale deployments
+and helps reduce operator error.
+.Pp
+Partial UUID calling is supported with every operation.
+For example,
+.Ar adae47cb-01a8-11e4-aa78-3c970ea3222f
+can be used in the form of
+.Ar adae47cb
+or just
+.Ar adae .
+In addition to partial UUID calling, jail TAGs can be used
+interchangeably.
+.Pp
+To ease jail identification, a TAG field is included in list mode which
+can be set to any string (hostname, label, note, etc.).
+If unset, the TAG field contains the creation date and time stamp by
+default.
+.Pp
+Properties are stored inside ZFS custom fields, eliminating the need
```

Comment:
This is no longer correct, the jail's properties are stored inside the jails dataset.
`/iocage/jails/UUID/config.json`. The part about it being easy to send is still correct as the file will move with the jail.